// const promise = new Promise((resolve, reject) => {
//   setTimeout(() => {
//     reject("error");
//   }, 3000);
// });

// promise
//   .then((token) => console.log(token))
//   .catch((err) => console.log(err))
//   .finally((x) => {
//     console.log(x);
//   });
